### ISSUE TITLE
Missing DPI option for Razer Basilisk Ultimate Receiver in 0.4.0

### ISSUE THREAD

**Issue by frapana** [1kc/razer-macos#329]
DPI customization was available on Razer Basilisk Ultimate Receiver in 0.3.7 but I cannot see it anymore in the menu on 0.4.0

> this might be a regression of [f9dedd7](https://github.com/1kc/razer-macos/commit/f9dedd7cc9ab3f79470bc4619341366892e36781)

**Comment by dhobi** [comment]
Are you sure it was part of the menu? AFAIK the DPI was only adjustable through the device settings window (click on the device name to open it). Are you maybe mixing it with the brightness menu entry which still exists?

**Comment by frapana** [comment]
@dhobi I reinstalled the previous version and I saw that it was 0.3.4, not 0.3.7. 
It had the old UI and DPI settings was available from "Mouse custom settings" menu item.
<img width="798" alt="0 3 4" src="https://user-images.githubusercontent.com/7498724/112817168-6e8fd400-9082-11eb-8b99-ec68a682788b.png">
I jumped too many versions and I did not notice that the UX changed so much.
Sorry, we can close this issue.
Thank you
